tests/lustre/typechecking/futurePass/test_add_tuples.lus

```diff
@@ -0,0 +1,7 @@
+node n (x, y: int) returns (p,q: int);
+(*@contract
+    guarantee p = q;
+*)
+let
+(p, q) = (x, y) + (y, x);
+tel
```

Comment:
This should go to futureFail.

Reply:
Well, this fails right now, so may be in `shouldFail` as that is the right behavior :)

Reply:
Yes, you are right.

Reply:
Fixed.